backend/endpoints/rom.py

```diff
@@ -214,11 +225,27 @@ async def get_rom_content(
     files_to_download = files or [r["filename"] for r in rom.files]
 
     if not rom.multi:
-        return FileResponse(path=rom_path, filename=rom.file_name)
+        return FileResponse(
+            path=rom_path,
+            filename=rom.file_name,
+            headers={
+                "Content-Disposition": f'attachment; filename="{quote(rom.file_name)}"',
+                "Content-Type": "application/octet-stream",
+                "Content-Length": str(rom.file_size_bytes),
+                "X-Accel-Redirect": f"/library/{rom.full_path}",
+            },
+        )
 
     if len(files_to_download) == 1:
         return FileResponse(
-            path=f"{rom_path}/{files_to_download[0]}", filename=files_to_download[0]
+            path=f"{rom_path}/{files_to_download[0]}",
+            filename=files_to_download[0],
+            headers={
+                "Content-Disposition": f'attachment; filename="{quote(files_to_download[0])}"',
+                "Content-Type": "application/octet-stream",
+                "Content-Length": str(rom.file_size_bytes),
```

Comment:
Good catch! I had the same issue trying to add the header to the zipped rom (too big or too small since we couldn't predict the output) and didn't include it. I'll fix this one as well.